src/main/resources/api.yaml

```diff
@@ -0,0 +1,174 @@
+swagger: '2.0'
```

Comment:
I don't like that this file is buried in `src/main/resources`. That seems to imply that it is somehow used by the source code. Have you found any other projects that keep swagger/OpenAPI files in the same repo as the code? I'm wondering if there's a standard way of doing this.

Reply:
Still exploring... Ideally, we'd want to generate the swagger file from the code, or generate the endpoint and model code from swagger.